prow/gerrit/reporter/reporter.go

```diff
@@ -157,11 +158,17 @@ func (c *Client) Report(pj *v1.ProwJob) ([]*v1.ProwJob, error) {
 		}
 		toReportJobs = append(toReportJobs, pjOnRevisionWithSameLabel)
 
+		succeeded := false
 		if pjOnRevisionWithSameLabel.Status.State == v1.SuccessState {
+			succeeded = true
 			success++
 		}
 
-		message = fmt.Sprintf("%s\nJob %s finished with %s -- URL: %s", message, pjOnRevisionWithSameLabel.Spec.Job, pjOnRevisionWithSameLabel.Status.State, pjOnRevisionWithSameLabel.Status.URL)
+		checkOrX := "❌"
+		if succeeded {
```

Comment:
nit: remove this and merge it into the upper if block

Reply:
done